auth-api/src/auth_api/schemas/basecode_type.py

```diff
@@ -24,7 +24,7 @@ class BaseCodeSchema(ma.ModelSchema):  # pylint: disable=too-many-ancestors, too
     class Meta:  # pylint: disable=too-few-public-methods
         """Maps fields to a default schema."""
 
-        fields = ('code', 'description', 'default')
+        fields = ('code', 'description', 'default', 'is_government_agency', 'is_business')
```

Comment:
Does this affect anything else? 

This is meant to be generic I think? `is_government_agency` is making it fairly specific

Reply:
I'd use CATTRS instead, we already included it 4 months ago

Reply:
thanks for pointing it out, you correct, I don't need it. I'll remove it

Reply:
This code does not look fun to use:

![image](https://github.com/bcgov/sbc-auth/assets/3484109/b5a3474f-267d-4715-9451-9dcdef49488e)

Reply:
I think it's going to be too much out of scope to change this, I'd just leave it for now... but in the future we should take this out